lib/timeline/fps.dart

```diff
@@ -23,18 +23,23 @@ class FramesChart extends LineChart<FramesTracker> {
     lastFrameLabel.element.style.top = '0';
     lastFrameLabel.element.style.bottom = null;
     lastFrameLabel.element.style.textAlign = '-webkit-right';
+    // Explicitly set height so tooltip does not overlap with frame tooltip.
+    lastFrameLabel.element.style.height = '30px';
```

Comment:
not your bug but it can be clever to do this sort of styling in dart with cascades. You can wrhite
lastFrameLabel.element.style
 .. top = '0'
 ..bottom = null
 .. textAlign = 'right'
  ..height = '30px';
That said, this styling looks like a hack we should probably just cleanup with CSS.
For example, setting bottom to null seems dubious.

Reply:
done.